dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldFilterParams.java

```diff
@@ -51,6 +52,8 @@
     @Builder.Default
     private final Set<String> filters = Collections.singleton( "*" );
 
+    private User user;
```

Comment:
`final`?

Reply:
It can't be, the initialization of this property can happen later in the pipeline (so we don't require a user property when creating a param object), keeping it non final for now.